cli.js

```diff
@@ -14,8 +14,7 @@ const humps = require('humps');
 
 const registries = require('./registries.json');
 const PKG = require('./package.json');
-const NRMRC = path.join(process.env.HOME, '.nrmrc');
-const NPMRC = path.join(process.env.HOME, '.npmrc');
```

Comment:
This line is deleted?

Reply:
Thanks, I fixed it.

Reply:
const NRMRC = path.join(process.env.HOME, '.nrmrc');
const NPMRC = path.join(process.env.HOME, '.npmrc');

const HOME = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'];